tuning/utils/preprocessing_utils.py

```diff
@@ -68,13 +115,13 @@ def validate_data_args(data_args: configs.DataArguments, packing: bool):
     # TODO(s) In future seupport two more formats:
     # 1. Allow no response template, and JSON with input/output fields and mask input
 
-    # 2. Allow pretokenized Dataset besides JSON.
-
 
 def get_data_collator(
```

Comment:
unit tests need to be added for updates. see https://github.com/foundation-model-stack/fms-hf-tuning/blob/6e836d14522de9092457cf63386ae6b20eaa2a2d/tests/utils/test_preprocessing_utils.py#L129 for reference. 
Probably after rebasing, we can re-use same unit test

Reply:
using the same unit test, thanks. 

But if you see some sort of duplicated flow in this function let me know, we will work it out, thanks